tcf_website/templates/reviews/new.html

```diff
@@ -4,75 +4,156 @@
 {% block title %}Write a Review - theCourseForum{% endblock %}
```

Comment:
This isn't really relevant to the PR, but shouldn't the name of this file be something like `new_review` or `review_form` for clarity? At the least, the name of the corresponding view is `new_review` so I think it should be consistent.

Reply:
I didn't create/edit this file, but I think it's fine because you will use it like `{% include "review/new.html" %}` or `return render(request, 'review/new.html')`.

Reply:
^yeah Brian named it but I agree with the thinking

If we want to update all the filenames (view/template/js/css) to be consistent, lmk.

Reply:
Updated all the files to new_review.[XXX]